### PR TITLE
Unit tests

### PR DESCRIPTION
**Pull request by nolancon**
Some more basic unit tests to help out with #6

### CODE REVIEW

File: pkg/display/allcoin/allCoin_test.go [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+		filteredRows [][]string
+	}{
+		{
+			allRows: [][]string{
+				{"", "BTC", "", "", "", "bitcoin"},
+				{"", "ETH", "", "", "", "ethereum"},
+				{"", "LTC", "", "", "", "litecoin"},
+				{"", "BTC", "", "", "", "bitcoin"},
+				{"", "BTC", "", "", "", "bitcoin"},
+			},
```
Comment (Gituser143) [inline pkg/display/allcoin/allCoin_test.go:39]: @nolancon maybe add a few cases where filter happens using the name of the coin? As of now the `filterRows()` function checks if the given filter string exists in either the symbol name or the name itself.

**Review by Gituser143 — CHANGES_REQUESTED** [review]

**Review by Gituser143 — APPROVED** [review]
This looks great! Unless you'd like to add more tests, I'll merge this PR.

**Comment by nolancon** [comment]
Cool. You can merge this as is, thanks.

**Comment by Gituser143** [comment]
@nolancon thank you for your contribution!

**Pull request merged**